common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -658,7 +700,7 @@ private static BlockEntry<ActiveBlock> createFireboxCasing(BoilerFireboxType typ
     public static final float[] RUBBER_LEAVES_DROPPING_CHANCE = new float[]{0.05F, 0.0625F, 0.083333336F, 0.1F};
 
     @Environment(value= EnvType.CLIENT)
-    public static BlockColor leavesBlockColor() {
+    private static BlockColor leavesBlockColor() {
```

Comment:
why private? now some addon has to copy this method if they want to use it

Reply:
Apologies, I thought it was only being used by GTBlocks.